EKF/airspeed_fusion.cpp

```diff
@@ -196,9 +196,6 @@ void Ekf::fuseAirspeed()
 		_state.quat_nominal.normalize();
 
 		// update covariance matrix via Pnew = (I - KH)P = P - KHP
-		float KH[_k_num_states][_k_num_states] = {};
```

Comment:
There was no need to declare the Kalman gain as a local variable and the class variable should be used to be consistent with usage elsewhere. This will have no effect on the functionality as the Kalman gain does not need to be persistent. A class variable was used to avoid time lost allocating an array each time.